helm-chart-sources/appscope/README.md

```diff
@@ -0,0 +1,75 @@
+![Cribl Logo](../../images/Cribl_Logo_Color_TM.png)
+
+# Appscope Helm Chart
+
+This Helm chart deploys [Appscope](https://appscope.dev/) in a Kubernetes Cluster.
+
+# Prerequisites
+
+1. Helm v3 installed – instructions are [here](https://helm.sh/docs/intro/install/).
+2. Cribl helm repo configured. App the repo locally with the following command:
+    `helm repo add cribl https://criblio.github.io/helm-charts/`
+
+
+# Configuration
+
+As built, this chart will deploy to a Kubernetes cluster an AppScope Deployment, Service, Mutating Webhook Admission Controller, Service Account, Cluster Role, Cluster Role Binding, and Secret for setting the AppScope configuration. 
+
+The required options to deploy this chart is the `cribl` when sending to a Cribl destination, or `metrics` and `events` when sending to non Cribl destinations.
+
+For all the available command line options, see [Values to Override](#values-to-override). 
+
+## Basic Configuration for Cribl Stream 
+
+To define a TCP-based connection to Cribl Stream on Cribl.Cloud, install the chart with the with the `cribl` config value defined as: `in.main-default-<organization>.cribl.cloud:10091`:
+
+ `helm install appscope cribl/appscope --set appscope.destination.cribl="tcp://in.main-default-<your-organization>.cribl.cloud:10091"`
+
+By default, Cribl.Cloud-managed instances of Cribl Stream have port `10091` configured to use TCP, and a built-in AppScope Source to receive data from AppScope. 
+
+To send metrics & events to a local [Stream](https://cribl.io/stream/) instance, deploy with chart with the `cribl` config value defined as: `cribl-internal:10090`:
+
+ `helm install appscope cribl/appscope --set appscope.destination.cribl="tcp://cribl-internal:10090"`
+
+## Basic Configuration for Cribl Edge 
+
+* To install the chart with the release name "appscope" and send metrics & events to [Cribl Stream](https://cribl.io/stream/) defined as: `cribl-internal:10090`:
```

Comment:
This section is for Cribl Edge, but links to Cribl Stream?

Reply:
Cribl Edge does not have any services listening by default, so you'll ideally need to explain how to enable the service on that chart. Not sure I recommend this approach...

Reply:
Ok, we'll remove it as we're planning on introducing Prometheus support later, which would make integrating with Edge easier.